### ISSUE TITLE
[Windows 10 UAP] Suggestion: Implement navigation with SplitView.

### ISSUE THREAD

**Issue by juanfranblanco** [PrismLibrary/Prism#38]
Implement navigation using SplitView including:
Building menus dynamically.
Creation of generic menu.
Contextual menu associated with specific View / Views.
Hierarchical menus.
with specific View / Views.
Hierarchical menus.

**Comment by bartlannoeye** [comment]
The SplitView control indeed provides a good entry point for dynamic menus. But I wonder if this should be a part of Prism, or be made available as an extension/plugin.

**Comment by juanfranblanco** [comment]
More abstract scenario could be: 

1 Menu creation static / dynamic, action enabling / disabling depending on context / state / view
2. Example / plugin SplitView Windows10 UAP
3. Example / plugin Xamarin..
4. WPF etc. (Ribbon?)

**Comment by arbsmith** [comment]
I have a Sample that derives from Jerry Nixon's SplitView example.  I modified it to fit in with Prism(WindowsStore).

**Comment by gep13** [comment]
@arbsmith is this same public? Thanks!

**Comment by arbsmith** [comment]
Take a look here:
https://github.com/arbsmith/Prism/tree/master/Samples/SplitView/SplitView

I am still working on Back Navigation.  And I had an implementation of Prism with Unity that I did not include but it would make the ViewModelLocator resolving better by using DI.

**Comment by briannoyes** [comment]
I believe we have this covered now with the addition of CreateShell method to allow you to create SplitView and put Frame in the content area. If not open a new issue with more specifics of what you are blocked on. Thanks.
